doc/Advanced.md

```diff
@@ -117,6 +117,32 @@ Simply call `ObjectsManager::addMetadata(...)`, like in
 ```
 This metadata will end up in the CCDB.
 
+## Access conditions from the CCDB
```

Comment:
Maybe it should be more explicit that there is one instance of CCDB for the QC objects and a different CCDB instance for the rest and that we have two different URLs for them.

Reply:
i have added a few more sentences, is it now clear?

Reply:
good